projects/RabbitMQ.Client/client/impl/Frame.cs

```diff
@@ -47,7 +48,7 @@ internal static class Framing
          * +------------+---------+----------------+---------+------------------+
          * | 1 byte     | 2 bytes | 4 bytes        | x bytes | 1 byte           |
          * +------------+---------+----------------+---------+------------------+ */
-        private const int BaseFrameSize = 1 + 2 + 4 + 1;
+        public const int BaseFrameSize = 1 + 2 + 4 + 1;
```

Comment:
would internal suffice?

Reply:
The Framing class is internal. I usually don't use internals in internal classes.